doc/development/development_guide.rst

```diff
@@ -157,6 +198,21 @@ Test coverage can be checked by running
 The output of the above command will show the coverage percentage of each
 file, as well as the line numbers of any lines missing test coverage.
 
+To obtain coverage, the ``pytest-cov`` plugin is needed.
+
+The coverage of a specific file can also be checked by generating a report:
+
+.. code-block:: python
+
+    pytest tests/backend/test_states.py --cov --cov-report=term-missing
+
+The previously mentioned ``pytest`` options can be combined with the
+coverage options. As an example, with the following command we can get the
+report of a specific file while also filtering out certain tests:
+
+.. code-block:: python
+
+    pytest tests/backend/test_states.py --cov --cov-report=term-missing -k 'not TestBaseGaussianMethods'
```

Comment:
:heart:

Reply:
Perhaps worth mentioning here that `-k` allows you to pass a boolean string using file names, test class/test function names, _and_ marks?

Reply:
Good idea! Added